__tests__/app.test.js

```diff
@@ -32,31 +32,65 @@ describe('app', () => {
             })
         })
     })
-        describe('/api/reviews', () => {
-            it('200 GET - responds with an array of review objects ordered by date', () => {
-                return request(app)
-                .get('/api/reviews').expect(200)
-                .then(({body}) => {
-                    expect(body.reviews).toBeInstanceOf(Array);
-                    expect(body.reviews).toHaveLength(13);
-                    body.reviews.forEach((review) => {
-                        expect(review).toMatchObject({
-                            owner: expect.any(String),
-                            title: expect.any(String),
-                            review_id: expect.any(Number),
-                            category: expect.any(String),
-                            review_img_url: expect.any(String),
-                            created_at: expect.any(String),
-                            votes: expect.any(Number),
-                            designer: expect.any(String),
-                            comment_count: expect.any(String),
-                        })
-                    })
-                    expect(body.reviews).toBeSortedBy('created_at', {
-                        descending: true
-                    })
+    describe('/api/reviews', () => {
```

Comment:
is this pull request for both reviews and reviews/:review_id ?

Reply:
no its just for reviews/:review_id. I'm not too sure why it showing both. i made a pull request for reviews yesterday and then merged it when i was given the go ahead by harrison.